ixdiagnose/plugin.py

```diff
@@ -0,0 +1,32 @@
+import concurrent.futures
+import os
+import traceback
+
+from .plugins.factory import plugin_factory
+from .utils.formatter import dumps
+from .utils.paths import get_plugin_base_dir
+
+
+def generate_plugins_debug() -> None:
+    os.makedirs(get_plugin_base_dir(), exist_ok=True)
+    with concurrent.futures.ProcessPoolExecutor(max_workers=3) as exc:
```

Comment:
Do we really need concurrent execution (that might affect debug contents)? Current debug gathers in <10 seconds even on a slow VM.